packages/plugin-ext/src/main/node/paths/plugin-paths-service.ts

```diff
@@ -95,7 +99,14 @@ export class PluginPathsServiceImpl implements PluginPathsService {
      * Generate time folder name in format: YYYYMMDDTHHMMSS, for example: 20181205T093828
      */
     private gererateTimeFolderName(): string {
-        return new Date().toISOString().replace(/[-:]|(\..*)/g, '');
+        const timeStamp = new Date().toISOString().replace(/[-:]|(\..*)/g, '');
+        // Ensures our timestamp generation logic is "valid".
+        // Changes to the timestamp structure may break old logs deletion logic.
+        debugger;
```

Comment:
Left over?

Reply:
Oops i was sure i deleted this :(

Reply:
Maybe i did not force push latest fix. Will check when next to laptop again